zephyr-make

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/make -rf
+
+BOARD ?= qemu_x86
+CONF_FILE = prj.conf
+
+TARGETS = run flash debug debugserver ram_report rom_report
+
+all: outdir/${BOARD}/zephyr.elf
```

Comment:
This breaks the use-case of having your source directory somewhere else than your build directory.